libs/go.json

```diff
@@ -192,6 +195,7 @@
   {"id":"go.yarf","imports":["github.com/yarf-framework/yarf"],"name":"YARF","repo":"yarf-framework/yarf","tags":["rest"],"tech":["go-web"],"status":"model-ready"},
   {"id":"go.zerver","imports":["github.com/cosiner/zerver"],"name":"Zerver","repo":"cosiner/zerver","tags":["rest"],"tech":["go-web"],"status":"model-ready"},
   {"id":"go.d3d9","imports":["github.com/gonutz/d3d9"],"name":"d3d9","repo":"gonutz/d3d9","tags":["directx"],"tech":["graphics-gaming"],"status":"model-ready"},
+  {"id":"go.pty","imports":["github.com/kr/pty"],"name":"pty","repo":"kr/pty","tags":["unix-terminals"],"tech":["gui"],"status":"awaiting-model"},
```

Comment:
tech == cli, right? it's terminals.

Reply:
rereading the repo, makes sense, not sure what made me think about gui